### PR TITLE
Fix typos and grammar

### PR DESCRIPTION
**Pull request by henrivdr**
# Description
This looks like a huge PR, but it is only fixes for typos and grammar.
I triple checked to make sure the msgids are in sync with the L"" strings in the code.
Comments, variable names and non English translation strings were excluded.

I learnt that typos were "fixed" before by using translations.  After this PR, those translations are no longer relevant and can be cleaned up.
Hopefully this sets a new baseline which will last us a while. :-)

## Tests
The code compiled and ran successfully.
The translation files were verified with ./run_gettext.sh

### CODE REVIEW

File: src/libslic3r/PrintConfig.cpp [hunk 1]
```diff
@@ -1534,7 +1534,7 @@ void PrintConfigDef::init_fff_params()
...
     def->category = L("Quality");
-    def->tooltip = L("Print sequence of the internal (inner) and external (outer) walls. \n\nUse Inner/Outer for best overhangs. This is because the overhanging walls can adhere to a neighouring perimeter while printing. However, this option results in slightly reduced surface quality as the external perimeter is deformed by being squashed to the internal perimeter.\n\nUse Inner/Outer/Inner for the best external surface finish and dimensional accuracy as the external wall is printed undisturbed from an internal perimeter. However, overhang performance will reduce as there is no internal perimeter to print the external wall against. This option requires a minimum of 3 walls to be effective as it prints the internal walls from the 3rd perimeter onwards first, then the external perimeter and, finally, the first internal perimeter. This option is recomended against the Outer/Inner option in most cases. \n\nUse Outer/Inner for the same external wall quality and dimensional accuracy benefits of Inner/Outer/Inner option. However, the z seams will appear less consistent as the first extrusion of a new layer starts on a visible surface.\n\n ");
```
Comment (henrivdr) [inline src/libslic3r/PrintConfig.cpp:1537]: I have no idea why this was detected as a change.   There was no changes to the string.
git CLI does not detect it as a change...

**Review by SoftFever — APPROVED** [review]
LGTM
Thank you

**Comment by SoftFever** [comment]
@henrivdr 
I've reverted this commit.

I noticed that the strings are only updated in the .pot/.po files, but not in the source code files (.cpp). Once we run run_gettext.bat --full, those changes will be replaced. Which resulted in losting all the translations in different language.

**Comment by henrivdr** [comment]
That' strange. I specifically checked that the strings in the source files are updated.
Can you see which source files are not updated?

**Comment by henrivdr** [comment]
Running ./run_gettext.sh -full shows no messages or changes to files.

**Comment by henrivdr** [comment]
Ok, I found missed ones.

**Comment by henrivdr** [comment]
@SoftFever the root cause was that I used _en.po as a reference instead of the .po.  
That resulted in some missed source files and also a **lot** of lines in the  .po files.
So we learn.
Please see new PR #6673 .

**Pull request merged**